clayui.com/src/templates/docs.js

```diff
@@ -362,5 +400,22 @@ export const pageQuery = graphql`
 				}
 			}
 		}
+		icons: mdx(fields: {slug: {eq: "docs/components/css-icons.html"}}) {
```

Comment:
Rather than making a _specific_ case for the css-icons file, could we make it work for all cases that use `.mdx` as a sibling?

Meaning it would be something like this, where it would compile any mdx file that is declared as a sibling.
```js
mdxTab: mdx(fields: {slug: {eq: $sibling}}) {
	code {
		body
	}
}
```

Additionally, we would probably want to update the existing `tabs` one by doing something like
```js
mdTab: markdownRemark(fields: {slug: {eq: $sibling}}) {
	html
}
```

Notice that we don't really need to find _all_ the tabs since we only ever declare a single css tab.

Reply:
And after this sort of change, we will have to do something like `const tab = mdTab || mdxTab;` in the logic above to get the actual tab to render. One of those tabs will render via its `html` and the other will render via its `code.body`